code/__DEFINES/xenoarchaeology.dm

```diff
@@ -0,0 +1,85 @@
+//Material defines
+///Safe
+#define XENOA_BLUESPACE /datum/xenoartifact_material/bluespace
+///Mild
+#define XENOA_PLASMA /datum/xenoartifact_material/plasma
+///Dangerous
+#define XENOA_URANIUM /datum/xenoartifact_material/uranium
+///Wildcard
+#define XENOA_BANANIUM /datum/xenoartifact_material/bananium
+///Artificial
+#define XENOA_PEARL /datum/xenoartifact_material/pearl
+///The gods are about to do something stupid
+#define XENOA_DEBUGIUM /datum/xenoartifact_material
```

Comment:
I feel debug shouldn't use parent type. itself must be an abstract type.
literally debugium mat.

Reply:
Nuh uh